api/src/main/resources/workbench.yaml

```diff
@@ -449,11 +449,11 @@ paths:
           schema:
             $ref: '#/definitions/ErrorReport'
 
-  /v1/workspaces/{workspaceNamespace}/{workspaceId}/notebook-list:
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/bucket-file-list:
     get:
```

Comment:
So I think what you had before -- an endpoint to retrieve just notebooks -- was fine. We don't need to fetch the config file name to the client.

Reply:
If we will not be showing this information in future on UI then yes there would be no point in increasing the traffic and bring in config files. I'll be reverting this and creating a new API that would be getting the data and localize it behind the scene.

Reply:
Sounds good!

Reply:
Fixed